content/docs/hooks-faq.md

```diff
@@ -208,6 +209,17 @@ There are a few more heuristics, and they might change over time as we fine-tune
 
 * `componentDidCatch` and `getDerivedStateFromError`: There are no Hook equivalents for these methods yet, but they will be added soon.
 
+### What is the flow of a function component that uses Hooks? {#hook-flow-diagram}
+
+This diagram visually explains, at a high level, the execution flow of a function component that uses Hooks.
```

Comment:
Can probably remove “, at a high level, “ and just say: “ the diagram below is a basic visual example showing the execution flow of a component using hooks.”

Reply:
@gaearon? thoughts? and is there anything else you need from me?